### DISCUSSION TITLE
My Project Ideas - Coopsand

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#56] [Ideas]
Hey Everyone,

Here are my ideas for the project:

- **Portfolio Piece:** I currently have a site set up via Google Sites and another site on Wix, but I think it'd be a huge accomplishment to build my own site to hand to employers.
- **CPR Website:** I am a CPR instructor, and am thinking about starting up a side job teaching classes. Having a website that people could sign up for classes on would be super helpful.
- **HEDI Website:** A site that goes into detail regarding Human Experience Design and Interaction. I've found that it is a little tricky to explain, and the description on USU's website isn't the best.
- **Therapy Website:** My aunt is a therapist and she is working on getting a website up and running.
- **Tears of the Kingdom Website:** I am obsessed with this game, so creating a website that has food recipes, shrine locations, etc. would be fun.
- **Book Blog**: I like to read, so creating a website where I can talk about books would be fun.

**Comment by kassyoveson** [comment]
These are all great ideas. Your ideas helped me think of project ideas. I agree that HEDI is very hard to explain to people. This is a new field that no one seems to know about. (Great job market for us when we are done) If you every do pursue the CPR website, I would suggest marketing to schools. The school I work at require a certain number of teachers who need to be certified in CPR.  Just a thought.
